lib/components/Database/DAO.ts

```diff
@@ -38,6 +40,20 @@ export async function update(table: string, query: Record<any, unknown>, data: R
 
 
 
+export async function validateTable(table) {
```

Comment:
This is an example of the promise question I had in slack. Should this validateTable promise resolve either a true or false? or resolve when true and reject when false?